src/ui_basic/textinput.cc

```diff
@@ -207,6 +207,8 @@ void AbstractTextInputPanel::Data::draw(RenderTarget& dst, bool with_caret) {
 }
 
 void AbstractTextInputPanel::layout() {
+	// Offset snaps the text panel to the start of input. Saved text always displays from beginning
+	d_->scrolloffset = get_text().length() / get_h();
```

Comment:
Since `scrolloffset` is about horizontal scrolling I don't understand why you divide by height here, can you explain?

Reply:
Not entirely sure why but the height adjust helps the text appear consistently across all window sizes. Just using the length of the string seems to show the text in almost the midpoint of the string. Using the width dimension causes the game to crash. The goal was to use number logic that would remain consistent.

Reply:
I believe the binaries have been remove correctly now